Dockerfile

```diff
@@ -6,9 +6,17 @@
 
 # Install base image and its dependencies
 FROM python:3.8-slim
+
+ENV DEBIAN_FRONTEND=noninteractive
+
 # hadolint ignore=DL3008, DL3013, DL3015
 RUN apt-get update && \
-    apt-get install -y vim-tiny && \
+    apt-get install -y  \
+    vim-tiny  \
+    krb5-config \
+    krb5-user \
+    libauthen-krb5-perl \
+    libkrb5-dev && \
```

Comment:
*question:* do we need to add Kerberos dependencies for Serial? If I am not mistaken, we add Kerberos to Snakemake because it supports Python and a way of modifying workflow based on it. In the case of Serial, it is not possible.

Maybe those dependencies are needed if the serial engine has a Kerberos sidecar?

Reply:
It looks like Kerberos dependencies are not needed. I will remove them from this PR.